dbms/src/Interpreters/ExpressionActions.cpp

```diff
@@ -306,12 +306,6 @@ void ExpressionAction::execute(Block & block) const
         break;
     }
 
-    case JOIN:
-    {
-        join->joinBlock(block);
```

Comment:
Clean up all Join logic in ExpressionAction Or
```
    case JOIN:
    {
        ProbeProcessInfo probe_process_info;
        join->joinBlock(block, probe_process_info);
        break;
    }
```

Reply:
> Clean up all Join logic in ExpressionAction Or
> 
> ```
>     case JOIN:
>     {
>         ProbeProcessInfo probe_process_info;
>         join->joinBlock(block, probe_process_info);
>         break;
>     }
> ```

done.